apps/common/utils.py

```diff
@@ -36,7 +38,7 @@ def get_student_attr(student):
             message="A user account already exists. Please check your email for details.",
             code=http_status.HTTP_409_CONFLICT
         )
-    return student.full_name, student.faculty_id
+    return student.last_name, student.first_name, student.faculty_id
```

Comment:
Should we use: First name and then Last name

Reply:
fixed